vpr/src/route/route_timing.cpp

```diff
@@ -1987,9 +1987,11 @@ bool is_iteration_complete(bool routing_is_feasible, const t_router_opts& router
     //in addition to routing being legal and the correct budgeting algorithm being set.
 
     if (routing_is_feasible) {
```

Comment:
I would re-order this if
Test the non-RCV (routing_budgets_algorithm != YOYO) first, so someone who is not thinking of RCV has less to read.  Then the rest of the code.